task_schedule.cfg

```diff
@@ -11,7 +11,7 @@ iterations 1
 
 data_dir     $ENV{SKA_DATA}/mica/      # Data file directory
 log_dir      $ENV{SKA_DATA}/mica/logs  # Log file directory
-bin_dir      $ENV{SKA_SHARE}/mica/     # Bin dir (optional, see task def'n)
+bin_dir      $ENV{SKA_ARCH_OS}/share/mica      # Bin dir (optional, see task def'n)
```

Comment:
I thought we decided SKA_ARCH_OS was deprecated so could tweak this https://github.com/sot/skare3/issues/22#issuecomment-412646934

Reply:
Indeed that is correct.  The only hesitation I find now (with experience a year later) is that using `$SKA` here is potentially confusing.  Yes, since this will be running in a production environment so `$SKA` will always work, but now that I mostly run in a standalone environment that *looks* confusing because I immediately think that `$SKA` is outside the installation environment where `share/mica` actually lives.  So maybe sticking with `$SKA_ARCH_OS` is not such a bad thing in this context because it gives an clear and distinct name to what we mean.

Reply:
I don't think it is very distinct?  Fine with this PR as-is but still confused this then.

Reply:
`$SKA_ARCH_OS` is distinct because it *only* exists a production environment and can only mean the root of the runtime environment directory.  This more clearly signals what's happening.  `$SKA` on the other hand is more ambiguous.  The Ska3 "always-true" definition of `$SKA` is where `$SKA/data` lives, but in a production env that happens to coincide with the root of the runtime env.

Reply:
We should probably bring this back to the SKA thread.  Why does SKA_ARCH_OS only exist in a production environment?  I didn't think flt_envs required that but it sounds like you have that convention in mind?

Reply:
Sorry, I've been sloppy with terminology, I was equating "production environment" with "sourcing flt_envs on HEAD linux", or maybe "not standalone".  Do you understand my statement now?

Reply:
Still not quite there.  SKA_ARCH_OS is defined in flt_envs.  Do you explicitly not use flt_envs in most of your work at this point and just set SKA manually, even though flt_envs theoretically could be run for any ska3 conda environment?  Should we move to equating use of flt_envs with a "production" environment?  

I see your point that it doesn't much matter because SKA_ARCH_OS should be defined in HEAD ska3/flight, I just didn't see that as quite distinct from any other ska environment just yet, but maybe that's just how I'm using things.

Reply:
I always set `export SKA=${HOME}/ska` and that's it.  That's where my Ska data live.  If you run `flt_envs` on a standalone environment that basically breaks Ska3 since `$SKA` is no longer pointing at your data (unless you do some shenanigans with a link).

Reply:
> We should probably bring this back to the SKA thread. 

That's done (in fact I did that before you suggested it!).

Reply:
Where?  I was wondering about updating the text in https://github.com/sot/skare3/issues/22

Reply:
https://github.com/sot/skare3/issues/22#issuecomment-412646934

But now I see the topic description needs tweaking to reflect the new approach on SKA_ARCH_OS.